src/inject/dynamic-theme/css-rules.ts

```diff
@@ -6,8 +6,17 @@ import {logWarn} from '../utils/log';
 export function iterateCSSRules(rules: CSSRuleList, iterate: (rule: CSSStyleRule) => void) {
     forEach(rules, (rule) => {
         if (rule instanceof CSSMediaRule) {
-            const media = Array.from(rule.media);
-            if (media.includes('screen') || media.includes('all') || !(media.includes('print') || media.includes('speech'))) {
+            const media: string[] = [];
+            Array.from(rule.media).forEach((mediaRule) => {
```

Comment:
The idea was that if there's `media.some()` used below, you will not have to loop through this array, search for ` and ` etc.

But OK, let's optimise it a bit. Use `let isScreenOrAll = false; let isPrintOrSpeech = false;`. Then loop through the records. If any starts with screen, all etc, then assign `true` for a corresponding boolean.

After that remove `media.some()`.